src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/Cells/CertificateReissuance/CertificateReissuanceTableViewCell.swift

```diff
@@ -0,0 +1,178 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import OpenCombine
+
+class CertificateReissuanceTableViewCell: UITableViewCell, UITextViewDelegate, ReuseIdentifierProviding {
```

Comment:
```suggestion
class CertificateReissuanceTableViewCell: UITableViewCell, ReuseIdentifierProviding {
```
I can't find where this gets used. If not needed we should remove it?